### PR TITLE
alanlima/c04-3m01

### PR DESCRIPTION
**Pull request by alanlima**
# Description

Exercise (e.g. C01-E01): **c04-3m01**

Student Name: **Alan Lima**

> Used 3M pattern to create/delete s3 buckets without need to install the awscli in your local machine

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/05class/exercises/c05-3m01/alanlima/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+create_bucket:
+	@$(RUNNER) make _create_bucket
+.PHONY:create_bucket
+
+delete_bucket:
+	@$(RUNNER) make _delete_bucket
+.PHONY:delete_bucket
+
+list_bucket_objects:
+	@$(RUNNER) make _list_bucket_objects
```
Comment (denstorti) [inline classes/05class/exercises/c05-3m01/alanlima/Makefile:13]: always going beyond, good work :)

File: classes/05class/exercises/c05-3m01/alanlima/Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	@scripts/delete_bucket.sh
+.PHONY:_delete_bucket
+
+_list_bucket_objects:
+	@echo "🤖: You again, I cannot even enjoy my standby moments in peace."
+	@echo "🤖: So now you want me to give you a list of the objects?"
+	@echo "🤖: Such a lazy human, just go to the aws console 😑. Fine here is the list:" 
+	@aws s3 ls --human-readable --summarize s3://${BUCKET_NAME}
+	@echo -e "\n🤖: anything else? maybe a mojito cocktail 🍸?"
+.PHONY:_list_bucket_objects
```
Comment (denstorti) [inline classes/05class/exercises/c05-3m01/alanlima/Makefile:30]: good one 🤣

**Review by denstorti — APPROVED** [review]
Good work, Alan! Funny chat with Mr. robot for managing buckets!

File: classes/05class/exercises/c05-3m01/alanlima/Makefile (hunk 2 above)
Comment (alanlima) [reply classes/05class/exercises/c05-3m01/alanlima/Makefile:30]: My creative moments 🤣🤣

File: classes/05class/exercises/c05-3m01/alanlima/Makefile (hunk 1 above)
Comment (alanlima) [reply classes/05class/exercises/c05-3m01/alanlima/Makefile:13]: Thanks, I like to play around with the cli. I missed it 😄

**Pull request merged**